.cloudbuild/library_generation/library_generation.Dockerfile

```diff
@@ -12,11 +12,26 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# install gapic-generator-java in a separate layer so we don't overload the image
+# with the transferred source code and jars
+FROM gcr.io/cloud-devrel-public-resources/java21 AS ggj-build
```

Comment:
Could we use image in Artifact Registry?

For example, https://airlock.corp.goog/package?type=Docker&id=088032b3-3669-43c0-9e82-02871abf2f0e

Reply:
I tried to scope this to baking the generator only. The next step in the milestone is to "cleanup" the Docker image, involving several improvements including using safer base images. Maybe we can leave it to the next task, WDYT?

Reply:
SGTM